environment.yml

```diff
@@ -7,56 +7,62 @@ channels:
   - conda-forge
   - mjohnson541
 dependencies:
-  - ase >=3.15.0
   - cairo
   - cairocffi
   - rmg::cantera >=2.3.0
-  - cclib >=1.6
+  - conda-forge::cclib >=1.7.0
   - rmg::chemprop
-  - codecov
   - coolprop
   - coverage
   - cython >=0.25.2
   - ffmpeg
   - rmg::gprof2dot
   - graphviz
   - h5py
-  - ipython
   - jinja2
   - jupyter
   - rmg::lpsolve55
-  - mako
   - markupsafe
   - matplotlib >=2.2.2
-  - mkdocs-material >=5.1.7
-  - mkdocs-material-extensions
-  - mock
   - rmg::mopac
   - mpmath
+  - rmg::muq2
   - networkx
-  - nose
   - rmg::numdifftools
   - numpy >=1.15.4
-  - openbabel
-  - paramiko >=2.6.0
+  - conda-forge::openbabel >= 3
   - pandas
   - psutil
-  - py3dmol >= 0.8.0
   - rmg::pydas >=1.0.2
   - pydot
   - rmg::pydqed >=1.0.1
   - pymongo
   - pyparsing
   - rmg::pyrdl
-  - pyrms
   - python >=3.7
-  - pytest
   - pyyaml
-  - qcelemental
   - rmg::quantities
   - rmg::rdkit >=2020.03.3.0
   - scikit-learn
   - scipy
-  - sphinx
   - rmg::symmetry
+  - xlrd
   - xlwt
+  - anaconda::sphinx_rtd_theme
+  - anaconda::paramiko >=2.6.0
+  - conda-forge::py3dmol >= 0.8.0
+  - conda-forge::ase >=3.15.0
+  - anaconda::ipython
+  - anaconda::sphinx
+  - conda-forge::qcelemental
+  - mako
+  - pytables
+  - pytorch::pytorch >=1.12.0
+  - anaconda::pytest
+  - conda-forge::torchani
+  - conda-forge::xtb
+  - mkdocs-material >=5.1.7
+  - mkdocs-material-extensions
+  - mock
+  - julia
```

Comment:
I agree. It might be a leftover, since we removed the RMS adapters and we run RMG in a subprocess within its own env. I'll remove `julia` from T3's env

Reply:
Maybe don't remove yet because as far as I can see when I attempt to run T3 locally, it won't switch env when running RMG files, so it needs julia still